autobahn/asyncio/websocket.py

```diff
@@ -194,18 +194,21 @@ class WebSocketServerProtocol(WebSocketAdapterProtocol, protocol.WebSocketServer
     def _onConnect(self, request):
         # onConnect() will return the selected subprotocol or None
         # or a pair (protocol, headers) or raise an HttpException
-        ##
-        # noinspection PyBroadException
-        try:
-            res = self.onConnect(request)
-            if yields(res):
-                asyncio.async(res)
-        except ConnectionDeny as e:
-            self.failHandshake(e.reason, e.code)
-        except Exception as e:
-            self.failHandshake("Internal server error: {}".format(e), ConnectionDeny.INTERNAL_SERVER_ERROR)
+        fut = txaio.as_future(self.onConnect, request)
+        fut.add_done_callback(self._on_connect_callback)
+
```

Comment:
Mmmmm. No (:. With `txaio` we need modify global `txaio.config.loop` before each call its api for working with multiple event loops. So now this pull request is not good.